client/src/views/about/ChangesModal.vue

```diff
@@ -38,7 +38,7 @@
                   {{ change.description }}
                 </ion-text>
                 <a
-                  :href="$t('app.projectSources') + '/issues/' + change.issue"
+                  :href="$msTranslate('app.projectSources') + '/issues/' + change.issue"
```

Comment:
Didn't know this was a thing. This should have it's own key in the translation files,
`$msTranslate('app.projectIssues', { issueId: change.issue });`

Granted, it's not currently used.